src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/BasePersonalisationProvider.java

```diff
@@ -152,6 +152,18 @@ public static void buildHearingRequirementsFields(AsylumCase asylumCase, Builder
             .build();
     }
 
+    public Map<String, String> getUploadAdditionalEvidencePersonalisation(AsylumCase asylumCase) {
```

Comment:
Looks like this is quite a specific method for uploadAdditionalEvidence.  Are we sure we want to put this in a Base class?  If we keep doing that then the class will eventually become gigantic.

Reply:
Decided to refactor this later in a more considered way.  Better to get this delivered now for business value.